miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDownloader.kt

```diff
@@ -21,15 +21,15 @@ import java.net.URL
 
 @Suppress("SwallowedException", "TooManyFunctions")
 internal class MiniAppDownloader(
-    private val storage: MiniAppStorage,
     private var apiClient: ApiClient,
-    private val miniAppStatus: MiniAppStatus,
-    private val verifier: CachedMiniAppVerifier,
+    initStorage: () -> MiniAppStorage,
+    initStatus: () -> MiniAppStatus,
+    initVerifier: () -> CachedMiniAppVerifier,
```

Comment:
I didn't want to pass the function parameters but I cannot update test cases when just put them in lazy properties.